IZ #14404: setup -net fails if workstation install present
Patch taken from CWS SRX644 setup09 branch

Index: oo_cvs/setup2/source/other/respfile.cxx
===================================================================
RCS file: /cvs/installation/setup2/source/other/respfile.cxx,v
retrieving revision 1.21
retrieving revision 1.21.34.1
diff -u -3 -p -r1.21 -r1.21.34.1
--- oo_cvs/setup2/source/other/respfile.cxx	27 Mar 2003 11:08:32 -0000	1.21
+++ oo_cvs/setup2/source/other/respfile.cxx	14 May 2003 10:31:20 -0000	1.21.34.1
@@ -529,6 +529,9 @@ USHORT ResponseFile::ProcessEvents()
                     break;
             }
 
+			if( eMode == INSTALL_NETWORK )
+				pEnv->SetLocal( FALSE );
+
 			ByteString aInstalledPath;
 			if( SiHelp::IsVersionAlreadyInstalled(aInstalledPath, pCScript, pEnv) )
 				Critical_Error( ERR_VERSIONALLREADYINSTALLED, "" );
@@ -544,9 +547,6 @@ USHORT ResponseFile::ProcessEvents()
                 nReturnEvent = EVT_START_UPDATE_OLD_VERSION;
                 break;
             }
-
-			if( eMode == INSTALL_NETWORK )
-				pEnv->SetLocal( FALSE );
 
 			if( eMode == INSTALL_WORKSTATION )
 			{
